modules/architecture-machine-roles.adoc

```diff
@@ -81,7 +81,7 @@ Extra controls apply to control plane machines to prevent you from deleting all
 [NOTE]
 ====
 ifndef::openshift-dedicated,openshift-rosa[]
-Exactly three control plane nodes must be used for all production deployments.
+Exactly three control-plane nodes must be used for all production deployments. However, on bare metal platforms, clusters that were installed by using the Assisted Installer can be scaled up to five control-plane nodes.
```

Comment:
The assisted installer installation method shouldn't be mandatory, however adding control plane nodes with the instructions from assisted seems to be the only way, does this make sense?

Reply:
In that case, I'll remove the mention of Assisted Installer here and in the other note in the cluster tasks file.